src/bin/clean.rs

```diff
@@ -0,0 +1,23 @@
+use std::io;
+use std::io::Write;
+use std::fs;
+use clap::{App, ArgMatches, SubCommand};
+use mdbook::MDBook;
+use mdbook::errors::Result;
+use get_book_dir;
+
+// Create clap subcommand arguments
+pub fn make_subcommand<'a, 'b>() -> App<'a, 'b> {
+    SubCommand::with_name("clean").about("Delete built book")
+}
+
+// Init command implementation
+pub fn execute(args: &ArgMatches) -> Result<()> {
+    use mdbook::errors::*;
+    let mut book_dir = get_book_dir(args);
+    book_dir.push("book");
```

Comment:
I don't think this respects when people use a different build directory. For example, you can set it [in your `book.toml`][1] or override it using [the `--dest-dir` flag][2]. Instead, you probably want to join the book root with either the [`config.build.build_dir`][3] or the `--dest-dir` argument (if manually specified by the user).

[1]: https://rust-lang-nursery.github.io/mdBook/format/config.html#build-options
[2]: https://rust-lang-nursery.github.io/mdBook/cli/build.html#--dest-dir
[3]: http://rust-lang-nursery.github.io/mdBook/mdbook/config/struct.BuildConfig.html#structfield.build_dir

Reply:
@Michael-F-Bryan, what do you mean by "join book root"?

Reply:
@Michael-F-Bryan, I tried my best, now it should work. Please test.